samples/bluetooth/direct_test_mode/prj.conf

```diff
@@ -4,13 +4,17 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-CONFIG_NCS_SAMPLES_DEFAULTS=y
+# Configure assertions
+CONFIG_ASSERT=y
+CONFIG_ASSERT_NO_COND_INFO=y
+CONFIG_ASSERT_NO_MSG_INFO=y
 
-# Use RTT logging
+CONFIG_HW_STACK_PROTECTION=y
```

Comment:
Most of the changes here are still not mentioned in the commit message.

Reply:
This doesn't change a previous sample configuration because hw stack protection and assertions were enabled by `CONFIG_NCS_SAMPLES_DEFAULTS`. If you think that I should describe it in commit message then I can do this @jfischer-no